src/LibraryManager/Manifest.cs

```diff
@@ -83,14 +85,16 @@ public static async Task<Manifest> FromFileAsync(string fileName, IDependencies
             return FromJson("{}", dependencies);
         }
 
+
         /// <summary>
         /// Creates an instance of the <see cref="Manifest"/> class based on
         /// the provided JSON string.
         /// </summary>
         /// <param name="json">A string of JSON in the correct format.</param>
         /// <param name="dependencies">The host provided dependencies.</param>
         /// <returns></returns>
-        internal static Manifest FromJson(string json, IDependencies dependencies)
+        [SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "Capturing exception detail")]
+        internal static (Manifest manifest, string diagnostics) FromJson(string json, IDependencies dependencies)
```

Comment:
How useful is it to pass up the diagnostics string?  We're logging details in the catch block here, and then almost immediately logging it again - well, `e.Message`, so not as much details but it's still redundant with the `e.ToString()` - and to the same loggers.  It'll be a verbose dump followed by a less verbose report of the same failure.

Can we keep the API simple (just return Manifest) and get enough info from what we're logging here?